docs/OperatorBestPractices.md

```diff
@@ -119,3 +119,7 @@ Each example makes use of one of the reference operators and includes
 instructions for deploying the reference operators to a cluster, either
 through the command line or client web console UI. The examples are
 available [here.](https://github.com/redhat-developer/service-binding-operator/blob/master/README.md#example-scenarios)
+
+## Service Binding Specification
+
+An effort has been taken to encapsulate the best practices of service binding in the form of a [specification](https://github.com/k8s-service-bindings). The goal of this specification is to create a Kubernetes-wide specification for communicating service secrets to applications in an automated way. It aims to create a mechanism that is widely applicable.
```

Comment:
IMHO, the last two sentence might be removed, without changing the message.

Reply:
At the same time, I would rather mention the spec in README.md, it is a more visible place.

Reply:
I don't have a strong opinion on this but I felt it would be good in Operator Best Practices as the spec is still being worked upon and it would take some time for us to incorporate the changes in SBO completely, like annotations part, annotations being extensions rather than in the core spec, etc. Till then maybe we can avoid it mentioning in the main README.md. ? cc @baijum @sbose78